workshops/management/commands/upgrade_instructor_profiles.py

```diff
@@ -41,7 +41,7 @@
         ('Programming with MATLAB', 'swcarpentry',
          'matlab-novice-inflammation', 'swc/matlab'),
         ('Data Organization in Spreadsheets', 'datacarpentry', 'excel-ecology',
-         'dc/spreadsheets'),
+         'dc/spreadsheet'),
```

Comment:
Yup, we should use the plural.  Once we've done this migration and 
updated the database with instructor info, we can scrub old migrations.